REACT/react-props/react-props_button/src/App.js

```diff
@@ -2,5 +2,26 @@ import React from "react";
 import "./styles.css";
 
 export default function App() {
-  return <h1>Replace me with your component!</h1>;
+  return (
+    <>
+      <Button
+        backgroundColor="black"
+        color="#f9f9f9"
+        disabled={false}
+        text="Click me"
+      />
+      <Button color="#ff0000" disabled={true} text="Don't click" />
+    </>
+  );
+}
+
+function Button({ backgroundColor, color, disabled, text }) {
+  return (
+    <button
+      style={{ color: color, backgroundColor: backgroundColor }}
```

Comment:
In JS, when the property has the same exact name (and case) as its value, you can simplify the syntax like this.

```suggestion
      style={{ color, backgroundColor }}
```